package-lock.json

```diff
@@ -4699,20 +4699,17 @@
       "dependencies": {
         "ansi-regex": {
           "version": "2.1.1",
-          "resolved": false,
-          "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
+          "bundled": true,
```

Comment:
Don't think I've seen this attribute before. What version of NPM are you running?

Reply:
Hmm, I was apparently on 6.14.11. Updated to 7.5.6. I still see "bundled" in package-lock.json, though.